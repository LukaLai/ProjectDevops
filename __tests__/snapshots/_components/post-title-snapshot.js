import { render } from '@testing-library/react'
import { PostTitle } from '../../../src/app/_components/post-title'

it('renders homepage unchanged', () => {
    const { container } = render(<PostTitle />)
    expect(container).toMatchSnapshot()
})