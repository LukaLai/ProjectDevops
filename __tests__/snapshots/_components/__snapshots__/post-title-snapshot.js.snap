// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <h1
    class="text-5xl md:text-7xl lg:text-8xl font-bold tracking-tighter leading-tight md:leading-none mb-12 text-center md:text-left"
  />
</div>
`;
